packages/multichain/src/adapters/caip-permission-adapter-eth-accounts.ts

```diff
@@ -136,12 +136,7 @@ export const setEthAccounts = (
       accounts,
```

Comment:
The comment on this function still reads:
```
* Additionally, this function  adds a `wallet:eip155` scope with empty methods, notifications, and accounts
 * to ensure that the `wallet:eip155` scope is always present in the caveat value.
 ...
 ```

Reply:
thank you for catching this https://github.com/MetaMask/core/pull/4978/commits/35ce68cce14d55ce28c5c3385c83391532b399ab